gdk-pixbuf/src/auto/pixbuf_animation.rs

```diff
@@ -137,28 +137,15 @@ impl PixbufAnimation {
     }
 }
 
-pub trait PixbufAnimationExt: 'static {
```

Comment:
Before merging this we should make sure that it's impossible for external code to implement these traits on anything. Otherwise we need to mark the trait unsafe or implement one of the many sealing mechanisms.

Reply:
I have opened https://github.com/gtk-rs/gtk-rs-core/issues/1093 to keep track of this task

Reply:
I think that is already happening as it requires `IsA<T>` and that is unsafe?

Reply:
Could we merge this now?